### PR TITLE
HPCC-16965 Performance warnings from Cassandra workunit code

### PR DESCRIPTION
**Pull request by richardkchapman**

### CODE REVIEW

**Comment by richardkchapman** [comment]
@ghalliday Please review

File: plugins/cassandra/cassandrawu.cpp [hunk 1]
```diff
@@ -3670,39 +3712,31 @@ class CCasssandraWorkUnitFactory : public CWorkUnitFactory, implements ICassandr
     unsigned validateRepository(bool fix)
     {
         unsigned errCount = 0;
-        // MORE - if the batch gets too big you may need to flush it occasionally
-        CassandraBatch batch(fix ? cass_batch_new(CASS_BATCH_TYPE_LOGGED) : NULL);
+        CIArrayOf<CassandraStatement> secondaryBatch;
```
Comment (ghalliday) [inline plugins/cassandra/cassandrawu.cpp]: minor: unused I think

**Comment by ghalliday** [comment]
@richardkchapman all looks good to me.  One comment about an unused variable.

**Comment by richardkchapman** [comment]
@ghalliday I have removed the unused variable and repushed

**Pull request merged**